pandas/core/arrays/datetimes.py

```diff
@@ -1011,6 +1011,34 @@ def date(self):
         'is_month_start',
         'is_month_start',
         "Logical indicating if first day of month (defined by frequency)")
+    """
+        Indicates whether the date is the first day of the month.
+        
+        Returns
+        -------
+        Series of bool
+            Series of boolean values indicating whether the date is the first
+            day of the month
```

Comment:
Can you add a period at the end

Reply:
@datapythonista done. Can this be merged/closed?